pkg/apis/resource/types.go

```diff
@@ -214,7 +214,18 @@ type BasicDevice struct {
 	// The maximum number of attributes and capacities combined is 32.
 	//
 	// +optional
-	Capacity map[QualifiedName]resource.Quantity
+	Capacity map[QualifiedName]DeviceCapacity
+}
+
+// DeviceCapacity describes a quantity associated with a device.
+type DeviceCapacity struct {
+	// Quantity defines how much of a certain device capacity is available.
+	//
+	// +required
+	Quantity resource.Quantity
```

Comment:
Should this field be named "Capacity" ?  it's a little duplicative, but "quantity" doesn't really convey anything extra.  Thoughts?

Reply:
"A device capacity consists of a quantity and potentially some additional, yet to be defined fields" sounds fine to me. No strong opinion. :shrug: 

cc @klueska

Reply:
What about Value?

Reply:
Works for me.

Reply:
I can live with `value`

Reply:
Renamed. Renaming this caused lots of conflicts, so I kept that change as a separate commit at the end.